lib/async.js

```diff
@@ -55,23 +54,23 @@ module.exports = function resolve (x, opts, cb) {
             else cb(new Error("Cannot find module '" + x + "' from '" + y + "'"))
         })
     }
-    
+
```

Comment:
please revert all the trailing whitespace removal on lines that you're not already changing